djsixpack/djsixpack.py

```diff
@@ -92,8 +107,31 @@ def convert(self, kpi=None):
         experiment_name = self._get_experiment_name()
         try:
             resp = session.convert(experiment_name)
+
+            if self.local:
+                try:
+                    participant = SixpackParticipant.objects.get(unique_attr=self.client_id, experiment_name=experiment_name)
+                    participant.convert = True
+                    participant.save()
+                except SixpackParticipant.DoesNotExist:
+                    pass
+
         except RequestException as e:
-            logger.exception("Error while trying to .convert: {err}".format(err=e))
+            logger.exception("Error while trying to .convert: %s" % e)
```

Comment:
Interesting - reading the comment in landscape, I agree with it. What it's saying, though, is that you should do `logger.exception("Error while trying to .convert: %s", e)`. 

Both the original and this change unconditionally do the substitution, but doing it like I have here only does the substitution if the logging is actually needed, for a minor performance improvement.